app/views/categories/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<section class="main-container">
+  <div class="headline">
+    <%= button_to 'Log out', destroy_user_session_path, method: :delete, class: "btn" %>
+    <h2>Categories</h2>
+    <%= link_to new_category_path do %>
+      <div class="arrow">&rarr;</div>
+    <% end %>
+
+  </div>
+
+  <div class="categories-container">
+    <% @categories.each do |category| %>
+      <% if can? :read, category %>
+        <div class="category-wrapper">
+          <div class="category-icon">
+            <img src="<%= category.icon %>" class="icon">
+          </div>
+
+          <div class="category-details">
+            <%= link_to category.name, category_path(category), class: "category-name" %>
+            <p class="category-date"><%= category.created_at.strftime('%Y-%b-%d') %></p>
+          </div>
+
+          <div class="total-amount">
+            <p>$<%= category.cash_flows.sum(:amount) %></p>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+
+  <div class="add-category">
+    <%= link_to "Add a new Category", new_category_path, class: "add-btn" %>
+  </div>
+</section>
```

Comment:
Just as suggested previously, let's also try to stick to the design for the transactions and category page. kindly make use of the right colors 😉.

yours
<img width="412" alt="image" src="https://github.com/anita00001/budget-app/assets/64643995/5e69fb14-3bb9-4939-84c1-468656097dab">

design
<img width="253" alt="image" src="https://github.com/anita00001/budget-app/assets/64643995/89942e22-2daf-402d-a23d-580fa25bc3b1">

Reply:
Thanks for the suggestion. I have made the necessary changes.